backend/src/interactions/updateUnconfirmedContribution/UnconfirmedContributionAdmin.role.ts

```diff
@@ -22,13 +23,36 @@ export class UnconfirmedContributionAdminRole extends AbstractUnconfirmedContrib
     )
   }
 
+  /**
+   *
+   * @returns true if memo, amount or creation date are changed
+   */
+  private isContributionChanging(): boolean {
+    if (this.wasUpdateAlreadyCalled()) {
+      throw new LogError('please call only before calling checkAndUpdate')
```

Comment:
I don't believe that this errormessage is expressiv for an user to know what he have to do, right?

Reply:
the same as previous.